pkg/document/block.go

```diff
@@ -102,12 +102,16 @@ func (b *CodeBlock) Background() bool {
 	return val
 }
 
-func (b *CodeBlock) Categories() []string {
-	cat, ok := b.Attributes()["category"]
+func (b *CodeBlock) Tags() []string {
+	tag, ok := b.Attributes()["category"]
+	if !ok {
+		tag, ok = b.Attributes()["tag"]
```

Comment:
We don't care about a case where both are set?

Reply:
Right, I am combining them instead.